github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -874,4 +874,64 @@ postsubmits:
           resources:
             requests:
               memory: "200Mi"
+    - name: post-project-infra-kubevirt-releasegraph
+      always_run: false
+      annotations:
+        testgrid-create-test-group: "false"
+      cluster: kubevirt-prow-control-plane
+      decorate: true
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+        workdir: true
+      labels:
+        preset-github-credentials: "true"
+      run_if_changed: "robots/cmd/perf-report-creator/shape.yaml"
+      spec:
+        containers:
+        - image: quay.io/kubevirtci/pr-creator:v20250320-f6c439c
+          env:
+          - name: GIMME_GO_VERSION
+            value: "1.22.5"
```

Comment:
I tihnk the repo is building with go v.124.4 now so this should probably be updated - https://github.com/kubevirt/project-infra/blob/cb801401698233401f4d50d0b7cd335f6c847366/go.mod#L237